src/app.js

```diff
@@ -0,0 +1,178 @@
+#!/usr/bin/env node
+'use strict';
+let fs = require('fs');
+let path = require('path');
```

Comment:
Todo lo que no cambie que sea const
```suggestion
const fs = require('fs');
const path = require('path');
```